include/swift/Strings.h

```diff
@@ -24,6 +24,8 @@ constexpr static const StringLiteral STDLIB_NAME = "Swift";
 constexpr static const StringLiteral SWIFT_ONONE_SUPPORT = "SwiftOnoneSupport";
 /// The name of the Concurrency module, which supports that extension.
 constexpr static const StringLiteral SWIFT_CONCURRENCY_NAME = "_Concurrency";
+/// The name of the Distributed module, which supports that extension.
+constexpr static const StringLiteral SWIFT_DISTRIBUTED_NAME = "_Distributed";
```

Comment:
I thought the module is called `Distributed` right now, with no underscore. The underscore is for implicitly-imported modules I think.

Reply:
The import is `_Distributed`, same as _Concurrency was and it did have to be imported explicitly for a while.